src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -662,14 +667,23 @@ def inner(*args, **kwargs):
             params = get_request_body_params(cherrypy.request)
             kwargs.update(params)
 
-            ret = func(*args, **kwargs)
+            req_version = '15'
+            if 'vnd.ceph.api.v' in cherrypy.request.headers.get('Content-Type'):
```

Comment:
I'd also extract `vnd.ceph.api...`. It's usually a good practice to extract literal data and reuse global constants.

```suggestion
            if cherrypy.request.headers.get('Content-Type').startswith(CEPH_API_CONTENT_TYPE):
```